ddtrace/contrib/internal/botocore/patch.py

```diff
@@ -59,6 +61,32 @@
 log = get_logger(__name__)
 
 
+def _load_dynamodb_primary_key_names_for_tables() -> Dict[str, Set[str]]:
+    try:
+        encoded_table_primary_keys = os.getenv("DD_BOTOCORE_DYNAMODB_TABLE_PRIMARY_KEYS", "{}")
```

Comment:
We should have a release note since this essentially releases a configurable feature for users.

We'll also want to update botocore/__init__.py with this configuration. We should probably have it configurable as both an environment variable and via code like the other configurations.